src/pages/postgraphile/background_tasks.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: page
+path: /postgraphile/background-tasks/
+title: Background Tasks in PostGraphile 
+---
+
+In your GraphQL server typically most processing will be done in the "frontground", as part of the request-response cycle. HTTP is a blocking protocol: once a client sends a request, it will wait for a reponse. This blocking nature requires the server to be fast at serving responses, to avoid keeping the client waiting for ever.
+
+But sometimes it is impossible to serve the response fast, for example we might need to do some heavy task like creating PDFs, calling an external API that takes a long time to respond, big calculations, or doing flaky tasks that need retries like sending an email. Or you might want to run some code periodically, detached from a user action. This are all good cases for background processing: the code runs outside of main request-response cycle, storing the result somewhere (typically on the database) and (optionally) notifying the client once it has finished.
+
+Task queues are systems that enable background processing for an application. They enable a way to persist this background processing tasks, notify the workers and handle errors and re-tries.
+
+### Tasks queues and workers for Postgres
+
+There are several exisiting solutions that can integrate with a PostgreSQL database or expose a Node.js-friendly API.
+
+- Graphile Worker
+- GCP Cloud Tasks
+- Faktory
```

Comment:
These two should have "(via Graphile Worker)" appended?

Reply:
Well, I was thinking of a structure for the docs like this
1. What are background tasks intro
2. General purpose libraries/solutions for background tasks
3. Graphile Worker intro
4. Combining Graphile worker with other libraries/solutions

What do you think of that order? Maybe we could simply do
1. What are background tasks intro
2. Graphile Worker intro
3. Combining Graphile worker with other libraries/solutions
4. Other libraries/solutions as reference (not supported with examples on how to integrate them)

Thoughts?

Reply:
Indifferent; the current order seems fine. I don't think you need to know what Graphile Worker is for the parens to make sense.